tests/test_ext_module.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+
+from pathlib import Path
+import pykokkos as pk
+
+
+# Tests if pk is able to compile an external module
+
+class TestExternalModule(unittest.TestCase):
+    def setUp(self):
+        self.path: Path = Path('./')
+        self.module_name = "test_ext_module"
+        self.source: List[str] = []
+
+        self.source.append("#include <pybind11/pybind11.h>\n")
+        self.source.append(f"PYBIND11_MODULE({self.module_name},m){{")
+        self.source.append(f"m.attr(\"__name__\") = \"{self.module_name}\";")
+        self.source.append("m.def(\"get_five\",[](){return 5;});")
+        self.source.append("}") 
+        self.ext_module = pk.runtime_singleton.runtime.compile_into_module(self.path,self.source,self.module_name,pk.ExecutionSpace.Default)
```

Comment:
Code is good, but do you think you can maybe add an extra layer of indirection somewhere under `interface/` that calls `compile_into_module` instead of having the user call it directly? Look at `interface/parallel_dispatch.py' to see what I mean

Reply:
done in https://github.com/kokkos/pykokkos/pull/119/commits/d5ddc1717d6cd91a01ec5943fecde2dc027cac07